### PR TITLE
Render Error Alert when update fails

### PR DESCRIPTION
**Pull request by wdrougas-tista**
Resolves [CASEFLOW-1925](https://vajira.max.gov/browse/CASEFLOW-1925)

### Description
Present user with alert when update of appellant fails due to a server failure

Screen Shot 2021-07-22 at 1 19 48 PM

### CODE REVIEW

File: client/app/queue/editAppellantInformation/editAppellantInformationSlice.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ const editClaimantSlice = createSlice({
 
 export const {
   cancel,
-  updateAppellantInformation,
```
Comment (wdrougas-tista) [inline client/app/queue/editAppellantInformation/editAppellantInformationSlice.js:20]: Missed this in the earlier PR

File: app/controllers/unrecognized_appellants_controller.rb [hunk 2]
```diff
@@ -9,6 +9,7 @@ def update
     else
       render json: unrecognized_appellant, status: :bad_request
     end
+  rescue StandardError
```
Comment (nanotone) [inline app/controllers/unrecognized_appellants_controller.rb:12]: Could you add a comment to this line, explaining why it's needed? Usually, blanket rescues are a code smell.

Comment (nweinmeister) [inline app/controllers/unrecognized_appellants_controller.rb:12]: We may want to allow this to actually fail. I'm not sure how our monitoring/logging works but typically if you rescue the actual error like this it won't surface to our monitoring which we may want. Also what gets returned for the API call with this?

Comment (wdrougas-tista) [reply app/controllers/unrecognized_appellants_controller.rb:12]: It returns `Internal Server Error`, I mocked failure by using faulty params in the controller. That makes sense though about logging, pushed up a change

**Review by nweinmeister — APPROVED** [review]
This looks good to go!

**Pull request merged**